odom/odom/launch/odom.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
+
+    <arg name="robot" default="caffeine"/>
+
+    <group if="$(eval arg('robot') == 'caffeine')">
+
+        <!-- Establish transform b/w Earth reference and Robot reference -->
+        <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform_node" respawn="true" ns="$(arg robot)">
+            <remap from="imu/data" to="imu/data"/>
+            <remap from="odometry/filtered" to="odom"/>
+            <remap from="gps/fix" to="gps/fix"/>
+
+            <rosparam command="load" file="$(find odom)/config/caffeine_navsat.yaml"/>
+        </node>
+
+    </group>
+
+</launch>
```

Comment:
One thing we should note is CAFF-8 also uses `odom.launch`. Should they both remain in the file or should we create two separate files (e.g. `local_odom.launch` and `navstat_odom.launch`)

Reply:
They should all remain in the same file, `odom.launch` should run all ekf and navsat nodes we need